app/connectors/EstatesStoreConnector.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.FrontendAppConfig
+import javax.inject.Inject
+import models.CompletedTasks
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class EstatesStoreConnector @Inject()(http: HttpClient, config : FrontendAppConfig) {
+
+  private val registerTasksUrl = s"${config.estatesStoreUrl}/register/tasks"
+
+  def getStatusOfTasks(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[CompletedTasks] = {
+    http.GET[CompletedTasks](registerTasksUrl)
+      .recover {
+        case _ => CompletedTasks()
+      }
+  }
```

Comment:
I realise this may have come from somewhere else, but do we really want to swallow all errors (e.g. network problems) and convert them to no tasks? If it's meant to handle the "no tasks yet" case, I wonder if it would make more sense to have the backend successfully return a default object (maybe it does).

Reply:
Yeah I think that's a fair comment, although `CompletedTasks()` doesn't refer to "no tasks", the apply method sets each task to false (i.e. incomplete).

Reply:
It does currently if there is no one found for the internal Id, it doesn't handle technical exceptions occuring as I would want to be alerted to that.
I think we could just allow it to bubble up the exception.